hatchery/helpers.go

```diff
@@ -151,6 +156,8 @@ func getAPIKeyWithContext(ctx context.Context, accessToken string) (apiKey *APIK
 	}
 
 	if resp != nil && resp.StatusCode != 200 {
+		b, _ := ioutil.ReadAll(resp.Body)
+		Config.Logger.Print(string(b))
```

Comment:
Is this logging intentional? I just want to make sure this wasn't left accidentally.